homeassistant/components/wled/const.py

```diff
@@ -30,3 +31,6 @@
 # Services
 SERVICE_EFFECT = "effect"
 SERVICE_PRESET = "preset"
+
+# Device classes
+DEVICE_CLASS_WLED_LIVE_OVERRIDE: Final = "wled__live_override"
```

Comment:
I've never seen a device class that looks like this. Can you shed some light on this?

Reply:
https://developers.home-assistant.io/docs/internationalization/core/#stringssensorjson